src/gt4py/gtscript.py

```diff
@@ -27,6 +27,7 @@
 import numpy as np
 
 from gt4py import definitions as gt_definitions
+from gt4py import ir as gt_ir
```

Comment:
This import should be avoided since it introduces a partial circular dependency between the `gtscript` and `ir` modules

Reply:
Ah ha!

> After discussion with @twicki, we agree that it is better to go back to use explicit mappings from DSL symbols to the NativeFunction IR enum, in order to achieve a better decoupling of the DSL and the IR.

Yes, this is concerning. Good idea to keep these separate.